packages/react-components/stable-review/react-components.api.md

```diff
@@ -37,6 +37,7 @@ export interface ActiveFileUpload {
     errorMessage?: string;
```

Comment:
Should this whole interface be not included in the stable api build?

Reply:
Removed it. :)